app/controllers/projects/classifications_controller.rb

```diff
@@ -0,0 +1,27 @@
+##
+# Controller responsible for handling project InfoRequest classification
+#
+# Requires `url_title` to be passed in as a param
+#
+class Projects::ClassificationsController < Projects::BaseController
+  include Classifiable
+
+  def create
+    set_described_state
```

Comment:
Sorry late review spot but this doesn't check that the user can read the project – we don't do that in `BaseController` right now

Reply:
Oh sorry, we check the project in `:update_request_state, so that's probably okay enough?

Reply:
Yeah I think that's enough for now.